### PR TITLE
header

### PR DESCRIPTION
**Pull request by pjmiles**
Header component created

### CODE REVIEW

File: src/components/api/axios.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import axios from "axios";
+
+export const imageURL = axios.get(
```
Comment (Asylcreek) [inline src/components/api/axios.js:3]: `imageURL` is not a worthy name.

It sounds more like a string than an axiosInstance

File: src/components/display/DisplayImage.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      setImages(res.data.results);
+      console.log(res.data.results);
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
+  useEffect(() => {
+    showImage();
+  }, []);
```
Comment (Asylcreek) [inline src/components/display/DisplayImage.js:20]: I feel `showImage` definition should be inside the `useEffect`

File: src/components/form/ImageForm.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
+import { useState } from "react";
+import "./ImageForm.css";
+
+const ImageForm = ({ closeModal }) => {
+  const [postImage, setPostImage] = useState({ title: "", file: "" });
+
+  const handleChange = (e) => {
+    e.preventDefault();
+    setPostImage({ ...postImage, [e.target.name]: e.target.value });
```
Comment (Asylcreek) [inline src/components/form/ImageForm.js:9]: Use the updater function

**Review by Asylcreek — CHANGES_REQUESTED** [review]

File: src/components/display/DisplayImage.js (hunk 2 above)
Comment (pjmiles) [reply src/components/display/DisplayImage.js:20]: ok

File: src/components/form/ImageForm.js (hunk 3 above)
Comment (pjmiles) [reply src/components/form/ImageForm.js:9]: ok

**Pull request merged**